chromium_src/chrome/updater/branding.gni

```diff
@@ -20,7 +20,7 @@ brave_updater_branding = {
   keystone_app_name = "BraveSoftwareUpdate"
   keystone_bundle_identifier = "com.brave.Keystone"
   mac_browser_bundle_identifier = "com.brave.Browser"
-  mac_updater_bundle_identifier = "com.brave.Updater"
+  mac_updater_bundle_identifier = "com.brave.BraveUpdater"
```

Comment:
this definitely doesn't cause any issues for existing installs?

Reply:
No, because this is Omaha 4 and isn't in prod yet.

Reply:
if these are specific to omaha 4 it would be better to put them behind the buildflag and also separate them out and add comments like omaha 3 below because it was not clear to me that these were not used in the current build

Reply:
They are used in the current build but hidden behind a default-disabled feature flag, which has not been enabled for any users.